client/src/locales/en-US.json

```
@@ -551,24 +551,24 @@
                 "cancelGreet": "Cancel"
             },
             "subtitles": {
-                "provideHostCode": "Click the code below on the guest.",
-                "getGuestCode": "Click on the code that appears on the guest.",
+                "provideHostCode": "Click the code below on the guest device.",
+                "getGuestCode": "Click on the code that appears on the guest device.",
                 "summary": "The device {label} has been added to your profile.",
-                "getDeviceInfo": "Waiting for you to fill the information on the guest.",
-                "waitForGuest": "You need to have Parsec open on both devices for the inscription duration.",
+                "getDeviceInfo": "Waiting for you to fill the information on the guest device.",
+                "waitForGuest": "You must have Parsec open on both devices for the inscription duration.",
```

"inscription" is not the right term here (does not means the same that in French). 
Should be "registration" or "onboarding" or "invitation" or "enrollment" or something like that.  
```suggestion
                "waitForGuest": "Parsec must be open on both devices during the onboarding process.",
```

---

I would like to go with onboarding or invitation, 'enrollment' is more of a military term.